c/include/libsbp/sbp.h

```diff
@@ -86,9 +86,18 @@ s8 sbp_remove_callback(sbp_state_t *s, sbp_msg_callbacks_node_t *node);
 void sbp_clear_callbacks(sbp_state_t* s);
 void sbp_state_init(sbp_state_t *s);
 void sbp_state_set_io_context(sbp_state_t *s, void* context);
+/* sbp_process has existed since dawn of sbp. It fires registered callbacks. */
 s8 sbp_process(sbp_state_t *s, s32 (*read)(u8 *buff, u32 n, void* context));
-s8 sbp_process_payload(sbp_state_t *s, u16 sender_id, u16 msg_type, u8 msg_len,
-    u8 payload[]);
+/* sbp_process_all allows a user's process_payload_fn to be called for every
+ * deframed sbp message. This is useful for logging or forwarding any message
+ * regardless of type. Optionally, it can call sbp_process_payload_cbs itself
+ * to invoke matching callbacks in the linked list as well.*/
+s8 sbp_process_all(sbp_state_t *s, s32 (*read)(u8 *buff, u32 n, void *context),
```

Comment:
I definitely prefer this naming scheme, but I agree with @silverjam that we shouldn't change the public API

Reply:
if need be we can marshall `sbp_process_payload` directly to `sbp_process_payload_cbs` with a comment stating that this api is deprecated.